packages/x-charts-vendor/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@mui/x-charts-vendor",
+  "version": "7.8.0",
+  "description": "Vendored dependencies for MUI charts",
+  "author": "MUI Team",
+  "main": "./index.js",
+  "keywords": [
+    "data visualization",
+    "React",
+    "d3",
+    "charting"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mui/mui-x"
+  },
+  "license": "MIT AND ISC",
+  "exports": {
+    "./package.json": "./package.json",
+    "./*": {
+      "types": "./*.d.ts",
+      "import": "./es/*.js",
+      "default": "./lib/*.js"
+    }
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.24.7",
```

Comment:
Am I missing something?
It seems that `@babel/runtime` is not used in any of the built files. 🤔 
Maybe we could move it to `devDependencies` and ignore in the package itself and also not commit into the repo? 🤔

Reply:
For a reason I ignore, the `test:regression` CI has some imports linked to it. I suspect their is a build with different configuration somewhere

https://app.circleci.com/pipelines/github/mui/mui-x/62610/workflows/97a60085-2198-4f6a-9547-34723031c5d2/jobs/357268

Reply:
I noticed all our packages have the `@babel/runtime` as a dependency, because us use it in our build process.

So I went the same with the vendor such that helpers are not inlined in the generated code. Will see if it solves the regression test